tests/performance_tests/test_import_time.py

```diff
@@ -69,12 +66,12 @@ def measure_import_time_in_ms(module_name: str, count: int = 3) -> float:
     """
     total_time = 0.0
 
-    for _ in range(count):
-        start_time = time.perf_counter_ns()
```

Comment:
using `perf_counter_ns` was intended to get more precious time measure, but admittedly for our case the error should be negligible

I personally prefer the original function name `measure_import_time_in_ms` as it's more obvious the time is given in ms not seconds.